lighthouse-core/scripts/generate-lh-timing-trace.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * @license Copyright 2016 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Technically, it's fine for usertiming measures to overlap, however non-async events make
```

Comment:
I'm not sure I understand overlapping vs async events here. Seems like they are separate concerns?